### DISCUSSION TITLE
zopen-setup - any option to just install a minimum set of tools?

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#366] [General]
The current zopen-setup installs a LOT of tools - many many never be used. Is there an option to just install the bare minimum set of tools?

**Comment by IgorTodorovskiIBM** [comment]
Not with zopen-setup, but once @DevonianTeuchter's [fork](https://github.com/DevonianTeuchter/meta) is made official, it will only install the bare minimum (curl and jq).

**Comment by MikeFultonDev** [comment]
@lbdyck What do you think of the new 'subset' of tools?
